### PR TITLE
Change the bucket cleanup script

### PR DESCRIPTION
**Pull request by OdedViner**

### CODE REVIEW

**Review by petr-balogh — DISMISSED** [review]

File: ocs_ci/utility/aws.py [hunk 1]
```diff
@@ -1902,37 +1902,56 @@ def list_buckets(self):
         """
         return self.s3_client.list_buckets()["Buckets"]
 
-    def get_buckets_with_prefix_(self, bucket_prefix, days):
+    def get_buckets_to_delete(self, bucket_prefix, hours):
         """
         Get the bucket with prefix which are older than given days
```
Comment (vavuthu) [inline ocs_ci/utility/aws.py:1907]:
```suggestion
        Get the bucket with prefix which are older than given hours
```

File: ocs_ci/cleanup/aws/cleanup.py [hunk 2]
```diff
@@ -312,19 +312,20 @@ def cluster_cleanup():
         p.join()
 
 
-def delete_buckets(bucket_prefix, days):
+def delete_buckets(bucket_prefix, hours):
     """
     Delete the S3 buckets with given prefix
 
     Args:
         bucket_prefix (str): Bucket prefix to delete.
```
Comment (vavuthu) [inline ocs_ci/cleanup/aws/cleanup.py:320]:
```suggestion
        bucket_prefix (dict): Bucket prefix as key and maximum hours to run/exist as value
```

File: ocs_ci/utility/aws.py [hunk 3]
```diff
@@ -1902,37 +1902,56 @@ def list_buckets(self):
...
+            bucket_delete_time = self.get_bucket_time_based_rules(
+                bucket_prefix, bucket_name, hours
+            )
+            # Get the creation date of the bucket in UTC
+            bucket_creation_date = bucket["CreationDate"].replace(tzinfo=timezone.utc)
+
+            # Calculate the age of the bucket
+            age_of_bucket = current_date - bucket_creation_date
+
+            # Check if the bucket is older than given days
```
Comment (vavuthu) [inline ocs_ci/utility/aws.py:1930]:
```suggestion
            # Check if the bucket is older than given hours
```

File: ocs_ci/cleanup/aws/cleanup.py (hunk 2 above)
Comment (OdedViner) [reply ocs_ci/cleanup/aws/cleanup.py]: done

File: ocs_ci/utility/aws.py (hunk 1 above)
Comment (OdedViner) [reply ocs_ci/utility/aws.py]: done

File: ocs_ci/utility/aws.py (hunk 3 above)
Comment (OdedViner) [reply ocs_ci/utility/aws.py]: done

**Review by vavuthu — APPROVED** [review]

**Review by petr-balogh — APPROVED** [review]

**Pull request merged**